smoke-tests-otel-starter/spring-boot-4/src/test/java/io/opentelemetry/spring/smoketest/OtelSpringStarterSmokeTest.java

```diff
@@ -49,8 +54,24 @@ void makeClientCall() {
   }
 
   @Override
-  void restClientCall(String path) {
+  void restTemplateCall(String path) {
     RestTemplate restTemplate = restTemplateBuilder.rootUri("http://localhost:" + port).build();
     restTemplate.getForObject(path, String.class);
   }
+
+  @Test
+  void restClientBuilder() {
+    testing.clearAllExportedData();
```

Comment:
why do you need to call `clearAllExportedData` here? In tests that use our `InstrumentationExtension` this gets automatically called before each test. If these smoke tests don't use it then perhaps it could be called from a `@BeforeEach` method so it wouldn't need to be called in each tests.

Reply:
one of the tests is asserting the telemetry emitted during the test setup, so BeforeEach couldn't be used, but `AfterEach` does